lib/Controller/Report.php

```diff
@@ -805,6 +808,19 @@ public function savedReportGrid(Request $request, Response $response)
             /** @var \Xibo\Entity\SavedReport $savedReport */
 
             $savedReport->includeProperty('buttons');
+            
+            $schemaVersion = $savedReport->schemaVersion;
+
+            // Show only convert button for schema version 1
+			if ($schemaVersion == 1) {
+
+                $savedReport->buttons[] = [
+                    'id' => 'button_convert_report',
+                    'url' => $this->urlFor($request,'savedreport.convert.form', ['id' => $savedReport->savedReportId] ),
+                    'text' => __('Convert')
+                ];
+                continue;
```

Comment:
`continue` because you don't want the rest of the buttons to be output?

I think an `else` would be much clearer here for someone coming to this at a later date.